src/batch-builder.js

```diff
@@ -349,6 +350,7 @@ module.exports = class BatchBuilder {
         this.input.toIdx[i] = tx.toIdx;
         this.input.auxToIdx[i] = tx.auxToIdx || 0;
         this.input.txCompressedData[i] = txUtils.buildTxCompressedData(Object.assign({newAccount: newAccount}, tx));
+        this.input.amountF[i] = float40.fix2Float(tx.amount || 0);
```

Comment:
Already parsed at the beginning when tx is added
```suggestion
        this.input.amountF[i] = tx.amountF;
```